pkg/download/jsoncreator/json_creator.go

```diff
@@ -101,7 +101,7 @@ func getDetailFromAPI(client rest.DynatraceClient, api api.Api, entityId string)
 func processJSONFile(data map[string]interface{}, id string) ([]byte, error) {
 	data = replaceKeyProperties(data)
 
-	jsonfile, err := json.MarshalIndent(data, "", " ")
+	jsonfile, err := json.MarshalIndent(data, "", "    ")
```

Comment:
Please remove this change. 
This was purposely not fixed yet, as it would mean all already downloaded files changing for users keeping them under version control. 
As such this was kept for a 2.x release, where things change substantially anyway (https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/issues/640)